cdap-ui/app/cdap/components/ThemeWrapper/colors.ts

```diff
@@ -1,6 +1,7 @@
 const colors = require('styles/colors.scss');
 
 export const blue = {
+  40: colors.blue07,
```

Comment:
should this be 600 instead of 40?

Reply:
So the reason I had this as 40 is because the least color is the dark shade of blue and this was darker than that guy. Added at 40 for consistent naming-to-shade map.